src/topology/algebra/uniform_group.lean

```diff
@@ -261,7 +267,7 @@ end
 
 end
 
-lemma to_uniform_space_eq {G : Type*} [u : uniform_space G] [add_comm_group G]
+lemma to_uniform_space_eq {G : Type u} [u : uniform_space G] [add_comm_group G]
```

Comment:
Why are you making this change? It's not needed right? Since `u` also denotes the uniform structure, I would prefer to keep the explicit universe out of the statement if possible.